packages/loaders/json-schema/src/getComposerFromJSONSchema.ts

```diff
@@ -743,6 +744,20 @@ export function getComposerFromJSONSchema(
               field: subSchema.discriminator.propertyName,
             },
           });
+
+          if (subSchema.discriminator.mapping) {
+            schemaComposer.addDirective(DiscriminatorMappingDirective);
+            Object.keys(subSchema.discriminator.mapping).forEach(value => {
+              const mappedSchema = subSchema.discriminator.mapping[value].split('schemas/')[1];
```

Comment:
Please tell me if there is a better way to do so, I've done it this way due to lack of context